leaderelection/leader_election.go

```diff
@@ -71,6 +83,8 @@ func NewLeaderElection(clientset kubernetes.Interface, lockName string, runFunc
 }
 
 // NewLeaderElectionWithLeases returns an implementation of leader election using Leases
+// healthCheckTimeout determines the max duration beyond lease expiration
```

Comment:
remove this comment since it's not specified here. ditto elsewhere

Reply:
updated